keystore/keyring.go

```diff
@@ -36,12 +36,16 @@ type TestKeyRingHolder struct {
 // KeyRing holds the keypair related to a specfic keypair type
 type KeyRing map[string]crypto.Keypair
 
+type ETHKeyRing map[string]*secp256k1.Keypair
+type SUBKeyRing map[string]*sr25519.Keypair
+
```

Comment:
Not sure why these are here

Reply:
This is my only concern. Otherwise PR LGTM